README.md

```diff
@@ -483,8 +483,47 @@ This is what the proposal evolved from. `async_hooks` in Node.js enabled async
 resources tracking for APM vendors. On which Node.js also implemented
 `AsyncLocalStorage`.
 
+## Chrome Async Stack Tagging API
+
+Frameworks can schedule tasks with their own userland queues. In such case, the
+stack trace originated from the framework scheduling logic tell only part of
+the story.
+
+```console
+Error: Call stack
+  at someTask (example.js)
+  at loop (framework.js)
+```
+
+The Chrome [Async Stack Tagging API][] introduces a new console method named
+`console.createTask()`. The API signature is as follows:
+
+```typescript
+interface Console {
+  createTask(name: string): Task;
+}
+
+interface Task {
+  run<T>(f: () => T): T;
+}
+```
+
+`console.createTask()` snapshots the call stack into a `Task` record. And each
+`Task.run()` restores the saved call stack and append it to newly generated
+call stacks.
+
+```console
+Error: Call stack
+  at someTask (example.js)
+  at loop (framework.js)
+  at async someTask               // <- Task.run
+  at schedule (framework.js)      // <- console.createTask
```

Comment:
I think this is at the wrong line? The `loop` function is the task item, right?

Reply:
The example [here](https://developer.chrome.com/blog/devtools-modern-web-debugging/#linked-stack-traces) shows that the async stack is appended to the current sync stack.

![image](https://user-images.githubusercontent.com/8500303/204423780-f9dd0fbb-164f-408a-a9ba-7283af5313b0.png)

Seems like this comment can be confusing. I've updated the comment to reflect that the async stack is appended after the `Task.run` in `loop`.